### PR TITLE
Kill Travis

### PR DESCRIPTION
**Pull request by zsmb13**
- Remove Travis config, as GitHub Actions is running the same tasks already
- Format GitHub Actions config, use official codecov action for coverage upload step

Further evidence of why this is a good idea:
image

### CODE REVIEW

**Review by JcMinarro — DISMISSED** [review]
:rocket: LGTM

**Comment by zsmb13** [comment]
@JcMinarro I think you'll also have to remove Travis CI from the list of required checks in the repository settings, because until then, we can't merge this PR 😄

**Comment by JcMinarro** [comment]
> @JcMinarro I think you'll also have to remove Travis CI from the list of required checks in the repository settings, because until then, we can't merge this PR

I already did it ;)

**Review by JcMinarro — APPROVED** [review]

**Pull request merged**
